### PR TITLE
Update tests to be able to run on XCode 12.5

### PR DESCRIPTION
**Pull request by arjankowski**
According to XCode 12.5 release notes a library libswiftXCTest.dylib (used by Nimble v8.0.0) is no longer included
and was replaced by libXCTestSwiftSupport.dylib. 

To make tests running on XCode 12.5 Nimble was updated to v9.0.1 which resulted in updating the API used in tests.

### CODE REVIEW

File: BoxSDK.xcodeproj/xcshareddata/xcschemes/TestApp.xcscheme [hunk 1]
```diff
@@ -0,0 +1,78 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (PJSimon) [inline BoxSDK.xcodeproj/xcshareddata/xcschemes/TestApp.xcscheme]: Just making sure: you want to commit this file?  Is it for the Test App you created recently or for the existing Test App in the project?

File: .idea/box-ios-sdk.iml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (PJSimon) [inline .idea/box-ios-sdk.iml]: Are these .idea files from IntelliJ?  Do we want to commit them for others to use?

**Review by PJSimon — DISMISSED** [review]
Left some comments (questions), but I'll go ahead and approve now, so as not to block you from merging, if all is good.

File: .idea/box-ios-sdk.iml (hunk 2 above)
Comment (arjankowski) [reply .idea/box-ios-sdk.iml]: I don't. I've already removed them in the next commit.

File: BoxSDK.xcodeproj/xcshareddata/xcschemes/TestApp.xcscheme (hunk 1 above)
Comment (arjankowski) [reply BoxSDK.xcodeproj/xcshareddata/xcschemes/TestApp.xcscheme]: This file is redundant. I've already removed it in the next commit.

**Review by PJSimon — APPROVED** [review]

**Pull request merged**